superlumic

```diff
@@ -133,10 +130,5 @@ if [ -f "config/$username.yml" ]; then
     setStatusMessage "Running the ansible playbook for $username"
     ansible-playbook -i "localhost," config/$username.yml
 else
-    if [ "travis" = "$username" ]; then
```

Comment:
I removed it because while your reasoning is sound, the travis build actually did not make use of the username "travis" as far as I can tell so this block doesn't really do anything.
Also, why not just let the travis build test `default.yml`? Seems quite sensible to me.